File: 28_02_2022/Find_Abundant_Mineral.py
n = int(input())
data = ""
for i in range(n):
    data += input()

maxx = 0
st = ""
for i in data:
    count = 0
    for j in data:
        if(i == j):
            count += 1
    if(count > maxx):
        maxx = count
        st = i
print(st)

File: February_2022/28_02_2022/Find_Abundant_Mineral.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

    int n;
    cin>>n;

    string s = "";
    for(int i=0;i<n;i++){
        string temp;
        cin>>temp;
        s += temp;
    }

    int maxx = 0;
    char d;
    for(int i=0;i<s.length();i++){
        int count = 0;
        for(int j=0;j<s.length();j++){
            if(s[i] == s[j]) count++;
        }
        if(count > maxx){
            maxx = count;
            d = s[i];
        }
    }
    cout<<d<<endl;

    return 0;
}